src/core/backend/pearl/disk_controller/logger.rs

```diff
@@ -0,0 +1,43 @@
+use super::*;
+use chrono::Local;
+use std::fs::{File, OpenOptions};
```

Comment:
I suggest to use tokio FIle. Or there is a reason to use sync version?

Reply:
The only thing, why I did this file sync is that we don't need to change interface of other function. If file is tokio, then we need `async new` (to create tokio File) -> `Pearl::new` becomes `async` -> `Backend::new` becomes `async` -> `Grinder::new` becomes `async`.
There is no problem in that, but disks events are so rare (I hope), that we won't see any difference between std and tokio `File`. But to keep `async` model of this lib I'll replace std `File` on tokio one

Reply:
One more possible solution to spawn runtime in one of this `new` methods (in `DisksEventsLogger`, for example) and make it sync, but that seems weird (it won't hit performance, because the creation of Logger is done once during application startup)